apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -77,7 +76,7 @@ handle_request('PostOracleQueryTx', #{'OracleQueryTx' := OracleQueryTxObj}, _Con
               <<"fee">>           := Fee} = OracleQueryTxObj,
             QueryTTLType = binary_to_existing_atom(maps:get(<<"type">>, QueryTTL), utf8),
             QueryTTLValue= maps:get(<<"value">>, QueryTTL),
-            case aec_base58c:safe_decode(oracle_pubkey, EncodedOraclePubkey) of
+            case aens:resolve(oracle_pubkey, EncodedOraclePubkey) of
```

Comment:
May be worth an integration of the description in the corresponding field in swagger.yaml